.github/workflows/pr-check.yml

```diff
@@ -15,7 +15,8 @@ jobs:
         run: |
           npm ci
           npm run lint
-          npm run build
```

Comment:
Is this because the `test` script is currently `npm run build && jest`, so `npm run build` is being run twice? I think this is ok as an optimization, however my concern would be that if the `test` script changes, then the CI is no longer running a build.

Reply:
well, if the test fails because of not running a build, we will notice 😅